apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/paidWorks.js

```diff
@@ -0,0 +1,157 @@
+        ({ iec, psrn, organizationAddress } = await financeInfoClient.getOrganization(organization.tin))
+    } catch (error) {
+        logger.info({ msg: 'fall financeInfoClient when get organization by tin', organizationId: organization.id, tin: organization.tin, error: error })
+    }
+    const { format, timeZone: timeZoneFromUser } = task
+
+    const timeZone = normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
+    const printDate = dayjs().tz(timeZone).locale(locale)
+
+    const contact = ticket.contact ? await getByCondition('Contact', {
+        id: ticket.contact,
+        deletedAt: null,
+    }) : null
+
+    const employee = organization.id && ticket.executor
+        ? await getByCondition('OrganizationEmployee', {
+            organization: { id: organization.id },
+            user: { id: ticket.executor },
+            deletedAt: null,
+        })
+        : null
+
+    const invoices = await Invoice.getAll(context, {
+        ticket: { id: ticket.id },
+        deletedAt: null,
+        status_not: INVOICE_STATUS_CANCELED,
+    }, 'id currencyCode accountNumber rows { isMin toPay name count vatPercent } recipient { bankName bankAccount bic }', {
+        sortBy: ['createdAt_ASC'],
+    })
+
+    const currencyCode = get(invoices, '0.currencyCode') || DEFAULT_INVOICE_CURRENCY_CODE
+    let totalSum = Big(0), totalVAT = Big(0)
+
+    const listOfWorks = invoices.reduce((acc, invoice) => {
+        const rows = Array.isArray(invoice.rows) ? invoice.rows : []
+        acc.push(...rows.map((row, index) => {
+            const price = Big(!row.isMin ? row.toPay : 0)
+            const sum = price.times(get(row, 'count', 1))
+            const vatPercent = Big(get(row, 'vatPercent', 0))
+            totalSum = totalSum.plus(sum)
+            totalVAT = totalVAT.plus(sum.times(vatPercent).div(100))
+
+            return {
+                number: index + 1,
+                name: row.name || '',
+                count: String(row.count) || '',
+                price: !Number.isNaN(price) ? renderMoney(price, currencyCode, locale) : '',
+                vat: !Number.isNaN(vatPercent) ? renderMoney(vatPercent, currencyCode, locale) : '',
+                sum: !Number.isNaN(sum) ? renderMoney(sum, currencyCode, locale) : '',
+            }
+        }))
+
+        return acc
+    }, [])
+
+    totalSum.toFixed(2)
+    totalVAT.toFixed(2)
+
+    const documentData = {
+        header: {
+            generalDate: printDate.format('DD.MM.YYYY'),
+        },
+        company: {
+            name: get(organization, 'name') || buildEmptyLineDifferentLength(true),
+            psrn: psrn || buildEmptyLineDifferentLength(),
+            tin: get(organization, 'tin') || buildEmptyLineDifferentLength(),
+            iec: iec || buildEmptyLineDifferentLength(),
+            address: organizationAddress || buildEmptyLineDifferentLength(true),
+            phone: get(organization, 'phone') || buildEmptyLineDifferentLength(),
```

Comment:
I'd have went the different way

1. companyData = { name, psrn, iec ... }
2. changeUndefinedDataToBlanks(companyData)

changeUndefinedDataToBlanks will just iterate over the object and if value is null or undefined it will simply use BLANK

you might want to store meta info about how long this is somewhere too